pkg/trace/api/otlp_test.go

```diff
@@ -747,6 +747,10 @@ func TestOTLPHelpers(t *testing.T) {
 				meta: map[string]string{"messaging.operation": "DO", "messaging.destination": "OP"},
 				out:  "DO OP",
 			},
+			{
+				meta: map[string]string{"messaging.operation": "DO", "messaging.destination.name": "OP"},
```

Comment:
I'm not sure if this scenario is possible, but we could add a test case for when both `messaging.destination` and `messaging.destination.name` are set. If this is possible, would we want to choose one over the other?

Reply:
Theoretically yes though in practice users should always use one semconv version. In that case the v1.6.1 semconv takes precedence (for backwards compatibility). I added a test for it.